src/opendune.c

```diff
@@ -1155,9 +1155,14 @@ static bool OpenDune_Init(int screen_magnification, VideoScaleFilter filter, int
 		return false;
 	}
 
+	g_screen_magnification = screen_magnification;
+	g_scale_filter = filter;
+
```

Comment:
Turned "screen_magnification" and "scale_filter" parameters into global extern variables, to can use them freely inside Video_Tick and Video_Init calls, since we added internal SDL_Init call into Video_Tick thread (see comments about Video_Init).